composer.json

```diff
@@ -12,47 +12,45 @@
     }
   ],
   "require": {
-    "php": "^7.2",
+    "php": "^8.1",
     "ext-json": "*",
     "ext-pdo": "*",
     "ext-sodium": "*",
     "ext-zip": "*",
     "barryvdh/laravel-debugbar": "^v3.6.7",
-    "barryvdh/laravel-dompdf": "^v1.0.2",
-    "doctrine/dbal": "^2.13.9",
+    "barryvdh/laravel-dompdf": "2.0.0",
+    "doctrine/dbal": "3.5.2",
     "fakerphp/faker": "^1.19.0",
-    "fruitcake/laravel-cors": "v2.2.0",
-    "laravel/framework": "^6.20.44",
+    "fruitcake/laravel-cors": "3.0.0",
+    "laravel/framework": "^9.0",
     "laravel/helpers": "^1.5.0",
-    "laravel/passport": "^9.4.0",
+    "laravel/legacy-factories": "^1.3",
+    "laravel/passport": "11.4.0",
     "laravel/tinker": "^2.7.2",
+    "laravel/ui": "^4.1",
     "maennchen/zipstream-php": "^2.1.0",
-    "moontoast/math": "^1.2.1",
     "neontribe/laravel-specification": "dev-master",
-    "ramsey/uuid": "^3.9.6",
-    "sebdesign/laravel-state-machine": "^2.1.1",
-    "symfony/event-dispatcher": "^4.4.37",
-    "symfony/console": "4.4.36",
+    "ramsey/uuid": "4.7.1",
```

Comment:
check this is actually needed as a top level dependency.
I think laravel may pull it in now.